server/pulp/server/async/scheduler.py

```diff
@@ -368,3 +368,7 @@ def add(self, **kwargs):
         entries to the database, and they will be picked up automatically.
         """
         raise NotImplementedError
+
+    def close(self):
```

Comment:
Can 1-liner docstring be added that indicates celerybeat calls this when closing.